kbs/config/kubernetes/ita/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: coco-tenant
+
+resources:
+- ../overlays/x86_64
+
```

Comment:
Just a note fur the future, but this doesn't block this PR.

I've asked @mythi about having the ITA image set here by default, but it seems that even if we have that the kustomization is not picking up the correct image.

We'll solve this and add the image here as a follow-up.